src/main/java/gmailgdogra/DtoWrapper.java

```diff
@@ -0,0 +1,16 @@
+package gmailgdogra;
+
+import java.util.List;
+
+public class DtoWrapper {
```

Comment:
The idea of a DTO wrapper is very new to me. Why do you need it?

Reply:
one of the easy way to return a list from Thymeleaf view. it be accessed in controller with 
@ModelAttribute DtoWrapper dtoWrapper

Reply:
Ok, got it